src/components/common/EventDialog.js

```diff
@@ -62,8 +63,11 @@ const EventDialog = ({ visible, event, onDismiss, reason }: EventDialogProps) =>
           >
             <Section style={{ backgroundColor: '#fff', marginBottom: '0' }}>
               <Section.Row>
-                <Avatar style={{ backgroundColor: '#777' }} />
-                <Text style={{ color: '#555' }}>From: {sender}</Text>
+                <Avatar size={48} style={{ backgroundColor: '#777', borderRadius: '50%' }} source={avatar} />
```

Comment:
inline styles are required here?

Reply:
@fernandomg , in [this PR](https://github.com/GoodDollar/GoodDAPP/pull/74) I replaced all inline styles by StyleSheet styles...in this branch I didn't check that and kept using same structure...anyway I can do the change also here